examples/getting_started/3_Deep_dive_into_the_anatomy_of_quantum_circuits/3_Deep_dive_into_the_anatomy_of_quantum_circuits.ipynb

```diff
@@ -1194,6 +1222,105 @@
     "print('Status of quantum task:', status)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## PARTIAL MEASUREMENT\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "<b>Note</b> The following code requires amazon-braket-sdk>=1.76.0 and amazon-braket-default-simulator>= 1.21.0\n",
```

Comment:
do we need to say this? I think the requirements.txt should enforce this already

Reply:
That's a good point. We probably don't need it since NBIs would have the latest version. My only thought is if someone was using this example locally they might not have the latest version, but that would be a pretty small amount of users. I'll go ahead and remove this.

Reply:
It's not a bad idea, but if we do this every time we add a new feature the notebooks would start to get pretty cluttered.